packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/TeamSettings.tsx

```diff
@@ -54,15 +53,15 @@ export const TeamSettings = () => {
               <RouterSwitch location={location}>
                 <Route
                   component={RegistrySettings}
-                  path={dashboardUrls.registrySettings()}
+                  path="/dashboard/settings/npm-registry"
```

Comment:
It looks good to me, but why don't we keep these URLs dynamic? It's seems easier to maintain if we have a single place to change it

Reply:
These will go away soon, the page should not render normally unless you access it by its old URL. and the dynamic functions are now returning portal URLs which don't work as route paths